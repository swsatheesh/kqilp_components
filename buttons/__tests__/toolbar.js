import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../button';
import Toolbar from '../toolbar';

jest.unmock('../button');
jest.unmock('../toolbar');

describe('components', () => {
  describe('<Toolbar />', () => {
    it('should render', () => {
      const toolbar = renderer.create(
        <Toolbar><Button /><Button /></Toolbar>
      ).toJSON();
      expect(toolbar).toMatchSnapshot();
      expect(toolbar.props.className).toEqual('btn-toolbar');
    });
  });
});
