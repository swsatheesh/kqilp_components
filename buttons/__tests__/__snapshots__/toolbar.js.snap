// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <Toolbar /> should render 1`] = `
<div
  className="btn-toolbar"
>
  <button
    className="btn btn-outline init-caps"
    disabled={false}
    onClick={undefined}
  >
    Click Me!
  </button>
  <button
    className="btn btn-outline init-caps"
    disabled={false}
    onClick={undefined}
  >
    Click Me!
  </button>
</div>
`;
